app/react/MetadataExtraction/SuggestionsAPI.ts

```diff
@@ -1,14 +1,27 @@
 import { RequestParams } from 'app/utils/RequestParams';
-import { SuggestionsSampleData } from './SuggestionsSampleData';
-
-const getSuggestions = async (requestParams: RequestParams) =>
-  // temporal sample data, should be replaced by real API call
-  new Promise(resolve => {
-    const data = SuggestionsSampleData(requestParams.data);
-    resolve({
-      suggestions: data,
-      totalPages: 30,
-    });
-  });
-
-export { getSuggestions };
+import api from 'app/utils/api';
+
+const getSuggestions = async (requestParams: RequestParams) => {
```

Comment:
For everything in this file: usually, our *API definitions in the front end go like this (example from PagesAPI).
```
  save(requestParams) {
    return api.post('pages', requestParams).then(response => response.json);
  },
```

From style perspective I don't think the difference is important, but if I read it correctly, the example returns a Promise and these do not. Is returning a Promise something we would like to keep to in our frontent API definitions?

Reply:
I'm not sure about this point. I think we don't have a convention for that, let's talk about it in the meeting

Reply:
Discussed in meeting, both ways are valid.

Reply:
Yes, I have misread it. Got to remember that async's return Promises automatically. :)